### PR TITLE
Match yeoman output

### PR DESCRIPTION
**Pull request by adrianwhite**
This tidies up the code to match that of the Yeoman generated pattern and updates the code to use Base.

### CODE REVIEW

**Comment by adrianwhite** [comment]
@jcbrand as you wrote the yeoman generator would you be able to please review this?
Cheers

File: src/pat-display-time.js [hunk 1]
```diff
@@ -1,58 +1,59 @@
...
+            "pat-logger",
+            "moment"
+        ], function() {
+            return factory.apply(this, arguments);
+        });
+    } else {
+        // If require.js is not available, you'll need to make sure that these
+        // global variables are available.
+        factory($, patterns.Base, patterns, patterns.Parser, patterns.logger);
+    }
```
Comment (jcbrand) [inline src/pat-display-time.js:19]: moment needs to be passed in here, like so:

`factory($, patterns.Base, patterns, patterns.Parser, patterns.logger, moment);`

File: src/pat-display-time.js [hunk 2]
```diff
@@ -1,58 +1,59 @@
...
+        trigger: ".pat-display-time",
+
+        init: function initUndefined () {
+            this.options = parser.parse(this.$el);
+            log.debug("pattern initialized");
+            this.processDate(this.$el, this.options);
         },
 
         processDate: function patDisplayTimeProcessDate($el, options) {
-            var dateStr = $el.attr('datetime');
```
Comment (jcbrand) [inline src/pat-display-time.js:45]: If you're using the Base pattern, then we have `$el` (and `options`) on the `this` parameter. So there's no need to pass them to `patDisplayTimeProcessDate`.

Instead, you can just get them from `this`, i.e. `this.$el` and `this.options`.

**Comment by jcbrand** [comment]
hi @adrianwhite, looks good, just two small things I mentioned in inline comments.

File: src/pat-display-time.js (hunk 1 above)
Comment (adrianwhite) [inline src/pat-display-time.js:19]: I thought that might have been the case. I meant to ask but it slipped my mind.

File: src/pat-display-time.js (hunk 2 above)
Comment (adrianwhite) [inline src/pat-display-time.js:45]: Ooo Base is very handy.

**Comment by adrianwhite** [comment]
@jcbrand I've pushed a couple of commits as per your notes.
Thank you very much for your input.

**Comment by jcbrand** [comment]
Cheers!

**Pull request merged**